flyway-core/src/main/java/org/flywaydb/core/internal/database/postgresql/PostgreSQLAdvisoryLockTemplate.java

```diff
@@ -109,7 +109,7 @@ private boolean tryLock() throws SQLException {
                 new RowMapper<Boolean>() {
                     @Override
                     public Boolean mapRow(ResultSet rs) throws SQLException {
-                        return "t".equals(rs.getString("pg_try_advisory_lock"));
+                        return rs.getBoolean(0);
```

Comment:
Column indexes are 1-based - otherwise this is a definite improvement!

Reply:
Couldn't we stick to using the column name instead of the index?